cdap-docs/user-guide/source/mmds/feature-gen.rst

```diff
@@ -0,0 +1,64 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2018 Cask Data, Inc.
+
+:hide-toc: true
+
+.. _user-guide-mmds-feature-gen:
+
+==================
+Feature Generation
+==================
+
+Before a model can be trained, features must be generated.
+This is done using the :ref:`Data Preparation Application <data-prep-user-guide-index>`
+Each row is a data point, and each column is a feature. Using the Data Preparation Application,
+you can get interactive feedback while you clean and transform your data source. In the example
+below, we have parsed our input, renamed columns, set various data types, and filtered out
+real estate prices that are under one million dollars.
+
+.. figure:: /_images/mmds-feature-gen.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+In this example, the outcome is price, and the features are city, zip, type, beds, baths, sqft, lot, stories, and year_built.
```

Comment:
`lot` --> `lot_sqft`?
You reference the column name as `lot_sqft` in https://github.com/caskdata/cdap/pull/10443/files#diff-f272ee6a7f9e505893c993c6dc040904R42